cirq-core/cirq/sim/act_on_args_container_test.py

```diff
@@ -226,6 +228,12 @@ def test_copy_succeeds():
     assert copied.qubits == (q0, q1)
 
 
+def test_copy_deprecation_warning():
+    args = create_container(qs2, False)
+    with pytest.warns(DeprecationWarning, match='reuse_buffer'):
```

Comment:
use `with cirq.testing.assert_deprecated`

Reply:
Got it. The code has been updated.